.github/workflows/container-image.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and Push Container Image
+
+on:
+  push:
+    branches:
+      - main
+      - buildanduploadimage
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Were you able to test this on your fork? I'll need to update our github secrets to acomodate this.